src/Client/Decorator/OAuth2/Token/AccessToken.php

```diff
@@ -66,10 +59,10 @@ public static function fromJsonString(string $json): self
         $data = json_decode($json, true);
 
         return new self(
-            isset($data['access_token']) ? (string) $data['access_token'] : '',
-            isset($data['expires_in']) ? (int) $data['expires_in'] : 0,
-            isset($data['token_type']) ? (string) $data['token_type'] : '',
-            isset($data['scope']) ? (string) $data['scope'] : ''
+            (string) $data['access_token'] ??= '',
+            (int) $data['expires_in'] ??= 0,
+            (string) $data['token_type'] ??= '',
+            (string) $data['scope'] ??= ''
```

Comment:
```suggestion
            (string) $data['access_token'] ?? '',
            (int) $data['expires_in'] ?? 0,
            (string) $data['token_type'] ?? '',
            (string) $data['scope'] ?? ''
```
eine Zuweisung zu den Properties des temporären Arrays `$data` erscheint mir wenig sinnvoll 🤷‍♂️

Reply:
Wow!, keine Ahnung wie das "=" da rein gerutscht  ist. Da fragt man sich warum der Interpreter einem das nicht um die Ohren haut.

Reply:
ist ja gültige Syntax, nur inhaltlich Blödsinn 😉 wäre das Array _by reference_ übergeben, würden diese Anweisungen ja sogar tatsächlich einen Effekt über ihre Zeile hinaus haben (nicht, dass die Zuweisung nicht auch dann oder sogar besonders dann fragwürdig wäre)